packages/cli/src/commands/run/index.ts

```diff
@@ -33,6 +33,7 @@ addLoggerOptions(runCommand)
   .option("--reset-cache", "resets the cache, filling it after a run")
   .option("--skip-local-cache", "skips caching locally (defaults to true in CI environments)", isRunningFromCI)
   .option("--profile [profile]", "writes a run profile into a file that can be processed by Chromium devtool")
+  .option("--ignore <ignore...>","ignores certain files when calculating the scope with `--since`", [])
```

Comment:
Need to update the help text here. Preferably something that expresses that this ignore is in addition to what's already ignored in the lage config.

Reply:
I think there needs to almost be a dedicated section in the doc on these flags!! But okay, I do think we need to wordsmith this a little bit!

Reply:
Done as suggested.